.circleci/config.yml

```diff
@@ -4,7 +4,18 @@
 # DOCKER_USER    - login info for docker hub
 # DOCKER_PASS
 #
-version: 2
+version: 2.1
+defaults:
+    docker_login: &docker_login
```

Comment:
this seems to just setup the docker_login "anchor" but doesn't invoke it?

Here's the [bmo example of invoking it](https://github.com/mozilla-bteam/bmo/commit/10bee2036db7c9c124815f84f91ab4c1beb55e4b)